### ISSUE TITLE
*take* parameter is impacting the sort of the results in a findmany query

### ISSUE THREAD

**Issue by Waldbillig-Thomas** [prisma/prisma#6095]
## Bug description

I wanted to create a __paginated__ resource that can be __sorted__ and __filtered__. and i noticed that the order of the results is varying depending on the value i give to the __take__ param

## How to reproduce

when changing the __take__ parameter (varying page size, not offset) I see the top of the list changing (and it should not)

~if i had to guess i would guess that the __take__ parameter is applied to the results before the __orderBy__ and which leads to incorrectly sorted results~

EDIT: It seems to be only when there are multiple identical values in the sorted column

## Expected behavior

the __take__ parameter should have no impact on a ordered list

## Environment & setup

- OS: Manjaro (Arch)
- Database: PostgreSQL
- Node.js version: v15.10.0
- Prisma version: 2.18.0

## tl;dr

do you need me to create some repo where the error can be reproduced?

b.t.w. thank you for the great work, it is really a great tool

**Comment by pantharshit00** [comment]
@Waldbillig-Thomas Yes, please create a reproduction for this so that we can see the problem and use that reproduction to fix the issue. Thanks!

**Comment by pantharshit00** [comment]
I am going to close this as its been 3 months now and there is no activity here. 

Please reply if you want us to take another look.
